doc/CMakeLists.txt

```diff
@@ -109,24 +124,96 @@ add_custom_target(
 )
 
 set(KI_SCRIPT ${ZEPHYR_BASE}/scripts/filter-known-issues.py)
+set(FIX_TEX_SCRIPT ${ZEPHYR_BASE}/doc/scripts/fix_tex.py)
 set(CONFIG_DIR ${ZEPHYR_BASE}/.known-issues/doc)
 
+#
+# HTML section
+#
 add_custom_target(
   html
   COMMAND ${CMAKE_COMMAND} -E env
   ZEPHYR_BUILD=${CMAKE_CURRENT_BINARY_DIR}
-  ${SPHINXBUILD} -w ${SPHINX_LOG} -N -t ${DOC_TAG} -b html ${ALLSPHINXOPTS} ${RST_OUT}/doc ${SPHINX_OUTPUT_DIR}
+  ${SPHINXBUILD} -w ${SPHINX_LOG} -N -t ${DOC_TAG} -b html -j auto ${ALLSPHINXOPTS} ${RST_OUT}/doc ${SPHINX_OUTPUT_DIR_HTML}
```

Comment:
Can't use the ``-j auto`` option for parallel builds (yet).  That's the cause of the current CI failure about ``WARNING: Duplicate declaration, also defined in...``

See https://github.com/michaeljones/breathe/issues/141

Reply:
Wow, I missed that. Thanks @dbkinder